### PR TITLE
CATTY-112 Show privacy policy on first launch

### PR DESCRIPTION
**Pull request by Bomml**
Everything is ready now for the Usability-Team. Only the links aren't clickable. I don't now how to fix that.

### CODE REVIEW

**Comment by m-herold** [comment]
Jenkins: retest this please

**Comment by m-herold** [comment]
Thank you very much for your PR! I will have a look at it asap. In the meantime please have a look at the UI tests, your changes involves changing some tests as well (see Jenkins log).

**Comment by Bomml** [comment]
Thank you Michael. I‘ll check.

**Comment by m-herold** [comment]
Jenkins: retest this please

File: src/Catty/ViewController/PrivacyPolicy/PrivacyPolicyViewController.swift [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+
+        scrollView.translatesAutoresizingMaskIntoConstraints = false
+        NSLayoutConstraint.activate([
+           scrollView.bottomAnchor.constraint(equalTo: acceptButton.topAnchor, constant: -16),
+           scrollView.leadingAnchor.constraint(equalTo: safeArea.leadingAnchor, constant: 8),
+           scrollView.trailingAnchor.constraint(equalTo: safeArea.trailingAnchor, constant: -8),
+           scrollView.topAnchor.constraint(equalTo: imageView.bottomAnchor, constant: -32)
+        ])
+        
+
```
Comment (GatCode) [inline src/Catty/ViewController/PrivacyPolicy/PrivacyPolicyViewController.swift]: because everything else looks very nice ;)

**Pull request merged**